packages/email/src/templates/new-message-from-program.tsx

```
@@ -0,0 +1,151 @@
+                    className="text-[13px] font-medium text-neutral-500 underline"
+                    href={`https://partners.dub.co/programs/${program.slug}`}
+                  >
+                    View program in Dub
+                  </Link>
+                </Section>
+              </div>
+            </Section>
+
+            <Section className="rounded-xl border border-solid border-neutral-200 p-6">
+              {messages.slice(0, MAX_DISPLAYED_MESSAGES).map((message, idx) => (
+                <Row className={idx > 0 ? "pt-3" : ""}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)